libs/src/main/python/dlpx/virtualization/libs/libs.py

```diff
@@ -468,7 +468,10 @@ def retrieve_credentials(credentials_supplier):
     response = internal_libs.retrieve_credentials(credentials_request)
     response_to_str(response)
     credentials_result = _handle_response(response)
-    if credentials_result.password != "":
+    if (
```

Comment:
nit: can you please add a comment to say why this check is needed and that way anyone modifying this part of code will know what to do/not do?

Reply:
Thank you Ravi. Sure, I've added the comment that explains about this change.